src/humanPose/HumanPoseRenderInstance.js

```diff
@@ -195,9 +195,12 @@ export class HumanPoseRenderInstance {
     }
 
     setVisible(visible) {
+        // MK HACK: too strict to do nothing if the visibility did not change. Other code can 'unhide' the slot
+        /*
```

Comment:
I believe this is a pretty sizeable performance loss but I'll do some proper testing to confirm. I definitely think this optimization is here for a reason

Reply:
The problem was that unwanted skeletons appeared in some cases. The slot of this render instance got changed (showSlot/hideSlot) but the visibility of this instance was not updated accordingly. Then the early exit did not give it a chance to correct itself.

But we spoke with @dangond-ptc and thought that one could just delete the instance object entirely instead for hiding it (by collapsing whole skeleton to 0,0,0).  There is a comment in function `hidePoseRenderInstance(poseRenderInstance) {...}`

Reply:
Yes, there's poseRenderInstance.remove() which will free the slot on the renderer. hidePoseRenderInstance could call remove() and set poseRenderInstances[poseRenderInstance.id] to null. I don't think this would fix whatever underlying behavior is causing this mismatch between poseRenderInstance.visible and the pose render instance's actual visibility.

Reply:
As promised, I did profile the impact of this change and it appears to be minimal (compared to all the other parts that need to be optimized). It would be better to fix the underlying issue but I'm okay with this hack for now

Reply:
Thanks.